src/modules.cpp

```diff
@@ -838,6 +839,25 @@ PVOID WINAPI DetourFindPayloadEx(_In_ REFGUID rguid,
     return NULL;
 }
 
+BOOL WINAPI DetourFreePayload(_In_ PVOID pvData)
+{
+	BOOL fSucceeded = FALSE;
+
+	HMODULE hModule = DetourGetContainingModule(pvData);
+	assert(hModule != NULL);
```

Comment:
The asserts would break a caller's error handling because it possibly changes GetLastError.

Reply:
The latest commit has resolved it.